htdocs/core/class/translate.class.php

```diff
@@ -90,6 +90,14 @@ public function setDefaultLang($srclang = 'en_US')
 
 		$this->origlang = $srclang;
 
+		if ($srclang == 'auto') {
+			if (!empty($mysoc->defaultlang)) {
+				$srclang = $mysoc->defaultlang;
```

Comment:
This will create a lot of confusion for user.
a german user with setup to "auto" will get the spanish language if company is spanish when he want german. Not what we want.
Also any change in behaviour must be pushed into develpo branch.

Reply:
Understandable, thank you for your answer.